scripts/hudson/narayana.sh

```diff
@@ -521,10 +522,17 @@ function rts_tests {
 function lra_tests {
   echo "#0. LRA Test"
   ./build.sh install -pl ArjunaCore/arjunacore,rts -am -DskipTests -Pcommunity "$@"
-  # we can't use 'mvn -f' option beacuse of Thorntail plugin issue THORN-2049
+  # we can't use 'mvn -f' option beacuse of Thorntail plugin issue QUARKUS-2049
   cd ./rts/lra/
+  echo "#0. Running LRA tests using $ARQ_PROF profile"
   PRESERVE_WORKING_DIR=true ../../build.sh -fae -B -Pcommunity -P$ARQ_PROF $CODE_COVERAGE_ARGS $ENABLE_LRA_TRACE_LOGS -Dlra.test.timeout.factor="${LRA_TEST_TIMEOUT_FACTOR:-1.5}" "$@"
-  [ $? = 0 ] || fatal "LRA Test failed"
+  lra_arq=$?
+  [ $lra_arq = 0 ] || echo "LRA tests using $ARQ_PROF profile failed"
+  echo "#0. Running LRA tests using $ARQ_QUARKUS profile"
+  PRESERVE_WORKING_DIR=true ../../build.sh -fae -B -Pcommunity -P$ARQ_QUARKUS $CODE_COVERAGE_ARGS $ENABLE_LRA_TRACE_LOGS -Dlra.test.timeout.factor="${LRA_TEST_TIMEOUT_FACTOR:-1.5}" "$@"
+  lra_thorn=$?
+  [ $lra_thorn = 0 ] || echo "LRA tests using $ARQ_QUARKUS profile failed"
+  if [ $lra_arq = 0 ] || [ $lra_thorn = 0 ] ; then fatal "LRA Test failed" ;  fi
```

Comment:
Oh, one more thing. I think this condition is wrong. The `0` means success and not `fatal`.

Reply:
yeah, just realised that...already changed and ready to be committed. Thanks Ondra